doc/hands_on/hands_on_dynamodb_vs.md

```diff
@@ -0,0 +1,229 @@
+1. [Create an access](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey) key for your AWS account
+1. create `~/.aws/credentials` and fill in:
+    ```
+    [default]
+        aws_access_key_id = <YOUR_ACCESS_KEY>
+        aws_secret_access_key = <YOUR_SECRET_ACCESS_KEY>
+        region = aws-region
+    ```
+   *Don't forget to replace `<YOUR_ACCESS_KEY>` and `<YOUR_SECRET_ACCESS_KEY>` by you keys from the previous steps.*
+
+1. Verify the setup by typing:
+    ``` shell script
+   aws help
+   ```
+
+### Local DynamoDB
+
+Amazon offers a local version of DynamoDB for testing purposes.
+You can run the local DynamoDB using plain Java, Maven or as a docker container (see [AWS documentation]((https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.DownloadingAndRunning.html))).
+In this guide we will use the docker version.
+You can however also use a different method.
+
+Steps for setup:
+ 
+1. Install DynamoDB:
+    ```shell script
+    docker run -p 8000:8000 amazon/dynamodb-local -jar DynamoDBLocal.jar -sharedDb -dbPath .
+    ``` 
+1. [Install AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+1. create `~/.aws/credentials` and fill in:
+   ```
+   [default]
+       aws_access_key_id = fakeMyKeyId
+       aws_secret_access_key = fakeSecretAccessKey
+       region = eu-central-1
+   ```
+   *The local DynamoDB does not check credentials. So we just need to add something for passing the checks in the CLI.*
+   
+In the following steps you have to append `aws help --endpoint-url http://localhost:8000` to all `aws dynamodb` commands. 
+
+
+## Setup sample data
+
+Now we are going to load some example data into the DynamoDB.
+For that we first of all need to create a DynamoDB table.
+We will do that suing the AWS CLI.
+For DynamoDB on AWS there is also a GUI.
+
+You can create the table by simply running the following command in a shell.
+
+``` shell script
+aws dynamodb create-table \
+    --table-name Books \
+    --attribute-definitions \
+        AttributeName=Title,AttributeType=S \
+    --key-schema \
+        AttributeName=Title,KeyType=HASH \
+    --provisioned-throughput \
+        ReadCapacityUnits=1,WriteCapacityUnits=1
+```
+
+You can verify that the table was created by running:
+
+```shell script
+aws dynamodb list-tables
+```
+
+The output should contain the `Books` table, we just created.
+
+### Load example data
+
+Now we will load the example data into the newly created table.
+
+Steps:
+
+1. [Download example data](./exampleData.json)
+2. Load the data into DynamoDB by running:
+
+``` shell script
+aws dynamodb batch-write-item --request-items file://./exampleData.json
+```  
+
+## Setup an Exasol database
+Now we need an Exasol database. In this guide we will use a local Exasol VM.
+Basically you can however select choose between the following options:
+
+* **Local Exasol VM (recommended)**
```

Comment:
I'm not sure if this description of Exasol's variants is necessary for this article